src/stores/notifications/RoomNotificationState.ts

```diff
@@ -24,16 +24,25 @@ import { Room } from "matrix-js-sdk/src/models/room";
 import * as RoomNotifs from '../../RoomNotifs';
 import * as Unread from '../../Unread';
 import { NotificationState } from "./NotificationState";
+import { isRoomMarkedAsUnread, UNSTABLE_MSC2867_MARKED_UNREAD_TYPE } from "../../Rooms";
+import SettingsStore from "../../settings/SettingsStore";
 
 export class RoomNotificationState extends NotificationState implements IDestroyable {
+    private featureMarkedUnreadWatcherRef = null;
     constructor(public readonly room: Room) {
```

Comment:
missing blank line

Reply:
linter is now happy https://github.com/matrix-org/matrix-react-sdk/pull/6314/commits/63f5ccbf158bbf152551c406688184f929cf5abb